modrob0:
  transformation_matrices: [
    1, 0, 0, 0, 
    0, 1, 0, 0, 
    0, 0, 1, 0.1, 
    0, 0, 0, 1, 
    ### 
    -2.051e-10, 1, 0, 0, 
    2.051e-10, 4.2067e-20, 1, -0.0655, 
    1, 2.051e-10, -2.051e-10, 0.2362, 
    0, 0, 0, 1, 
    ### 
    -2.051e-10, -1, 4.1021e-10, -7.1786e-11, 
    -1, 2.051e-10, -8.4135e-20, -0.35, 
    0, -4.1021e-10, -1, 0.1353, 
    0, 0, 0, 1] 

  tool_transformation_matrix: [
    1, 0, 0, 0,
    0, 1, 0, 0,
    0, 0, 1, 0.119550,
    0, 0, 0, 1]

  enclosures: [
    0.00044198, -0.046902, 0.2161,
    0.00056332, -0.039613, 0.20926,
    0.12876,
    ###
    -0.00068635, -0.31377, 0.11657,
    0.00068645, -0.02441, 0.13581,
    0.12102,
    ###
    -4.5649e-05, -9.1593e-16, 0.12055,
    4.5649e-05, 9.1593e-16, 0.27055,
    0.046807]

  nb_joints: 3